angular-rails-app/db/migrate/20170225185334_init_schema.rb

```diff
@@ -123,13 +123,13 @@ def up
       t.string   "zipcode",             limit: 255
       t.integer  "total_units",         limit: 4
       t.string   "bin",                 limit: 255
-      t.boolean  "rent_stabilized"
+      t.integer  "rent_stabilized",     limit: 4
```

Comment:
Yep, let's get this merged. I think we're on the same page about the long term plan is, and what to do in the interim.

If you're curious, there's an [ignore_tables](https://apidock.com/rails/v3.1.0/ActiveRecord/SchemaDumper/ignore_tables/class) option in ActiveRecord. I played with it, to see if would address your issues, but there were weird interactions with the `table_prefix`.